.travis.yml

```diff
@@ -10,6 +10,6 @@ rvm:
 
 script:
 - cd service_management/azure && bundle install && bundle exec rake test:unit
+- if [ "$TRAVIS_ASM_INTEG_RECORDED" == "true" ] ; then bundle exec rake test:recorded:affinity_group ; fi
```

Comment:
Cool, for some reason I couldn't see it in the settings for Travis, but now looks like I do. Thanks!